modules/core/arrow-core-extensions/src/main/kotlin/arrow/core/extensions/option.kt

```diff
@@ -27,6 +27,7 @@ interface OptionSemigroup<A> : Semigroup<Option<A>> {
 
 @extension
 interface OptionSemigroupal : Semigroupal<ForOption> {
+  override fun <A> id(): Option<A> = Option.empty()
   override fun <A, B> Kind<ForOption, A>.product(fb: Kind<ForOption, B>): Kind<ForOption, Tuple2<A, B>> =
```

Comment:
Nit: whitespace

Reply:
Which whitespace do you mean?

Reply:
Breaking line in between 29 and 30

Reply:
I can do that, but isn't the current format like in other functions (`OptionMonoid`, `OptionApplicativeError`, ...)?

Reply:
Oh, if that's the case nevermind. It's a nit anyway.